TestFoundation/TestFileManager.swift

```diff
@@ -770,6 +771,80 @@ class TestFileManager : XCTestCase {
         }
         XCTAssertNil(try? fm.linkItem(atPath: srcLink, toPath: destLink), "Creating link where one already exists")
     }
+    
+    func test_resolvingSymlinksInPath() {
+        do {
+
+            // Initialization
+            var baseURL = FileManager.default.temporaryDirectory.appendingPathComponent("test_resolvingSymlinksInPath")
```

Comment:
I think we have some of our other tests here stick a UUID in the path, so that if something gets stuck around from a previous run it doesn't hose the next test.

Reply:
Yes, good idea. Added!